Taskfile.yml

```diff
@@ -34,6 +34,7 @@ tasks:
       - echo "#OTEL_SDK_DISABLED=true" >> deployment/local-dev/local.env
       - echo "GOOGLE_OAUTH_CLIENT_ID=__REPLACE__.apps.googleusercontent.com" >> deployment/local-dev/local.env
       - echo "GOOGLE_OAUTH_CLIENT_SECRET=__REPLACE__" >> deployment/local-dev/local.env
+      - kubectl --context=docker-desktop apply -f deployment/setup/namespace.yaml
```

Comment:
We need to now explicitly create the namespace here?
So, how does this work in environments that don't run this setup (e.g. prod)

Reply:
Well, presumably the languagedepot namespace isn't being deleted and re-created in prod. But I can add the namespace creation back into the production and staging deployments if you think there's a chance of its being deleted.

But my thinking about k8s namespaces is that they're set up by the devops team, and deployments should assume namespaces already exist rather than creating them. Only reason I had a k8s resource creating the namespace was for local dev, where it's likely the docker-desktop k8s cluster is brand new with no namespaces in it yet.

Reply:
OK, seems reasonable to me.

Reply:
can confirm the namespace isn't being created in production (checked the deploy repo), and actually the namespace file wasn't referenced anywhere anyway from what I can tell so I'm not really sure how this fixes the issue

Reply:
The changes to `ingress-deployment.yaml` are what actually fix the issue. But this Taskfile.yml change is necessary for new devs to be able to run `skaffold dev` and not have it error out on first run.